ui/src/main/java/org/pentaho/mantle/client/workspace/SchedulerUiUtil.java

```diff
@@ -72,7 +72,7 @@ private SchedulerUiUtil() {
   includes duplicated values, so we remove those too. Also converts the params array into
   a more convenient list type.
    */
-  public static List<JsJobParam> getFilteredJobParams( JsJob job ) {
+  public static List<JsJobParam> getFilteredJobParams( JsJob job, final boolean hideInternalVariables ) {
```

Comment:
@srallapa We are adding a parameter to a public static method. Did you verify the usages of this method to ensure all pass 2 parameters instead of 1?

Reply:
Yes I have verified the entire pentaho-scheduler-plugin and I see only these are the places its been used.